force-app/main/default/lwc/picklist/picklist.js

```diff
@@ -22,9 +22,15 @@ export default class Picklist extends LightningElement {
         }
 
         this._options = this.picklist.picklistValues.map(picklistValue => {
-            // TODO: Set defaultValue if value not provided?
             let isSelected = this.value && this.value.includes(picklistValue.value);
 
+            if (
```

Comment:
I'm not sure this is working as intended. If picklistValue.defaultValue and !this.value would we just want to set this.value to picklistValue.defaultValue?

Reply:
I think I was testing something and left it in. Sorry I removed it. Thanks for catching it